tensorflow/tools/xla_extract/tf_graph_to_xla_lib.cc

```diff
@@ -42,25 +43,124 @@
 
 namespace tensorflow {
 
+namespace {  // anonymous namepace
+
+bool is_true(const char *s) {
+    if (s && *s) {
+        const char c = ::tolower(*s);
+        if (c == 'y' || c == 't') {
+            return true;
+        }
+        return atoi(s) > 0;
+    }
+    return false;
+}
+
+bool get_env_bool(const char *s, const bool dflt) {
+    const char *v = getenv(s);
+    if (v && *v) {
+      return is_true(v);
+    }
+    return dflt;
+}
+
+int get_env_int(const char *s, const int dflt) {
+  const char* v = getenv(s);
+  if (v && *v) {
+    return atoi(v);
+  }
+  return dflt;
+}
+
+const bool save_messages = get_env_bool("XLA_SAVE_MESSAGES", false);
+const bool verbose = get_env_int("XLA_LOG", NO_LOG) >= DEBUG_LOG;
```

Comment:
can we update `xla_log >= DEBUG_LOG` to use this? there should be some which use INFO_LOG, we can create another bool for that as well, or can leave them as is.

Reply:
done